CONTRIBUTING.md

```diff
@@ -430,6 +432,8 @@ continuous integration system. Interacting with Jenkins is not usually
 necessary, as Jenkins automatically picks up new patch sets and posts
 the results. Typically the only interaction needed with Jenkins is
 when some kind of glitch occurs and a build needs to be retriggered.
```

Comment:
glitch is a bit vague, maybe mention the example of spurious test failures or others